WeakAurasOptions/OptionsFrames/Update.lua

```diff
@@ -1807,6 +1829,14 @@ local methods = {
       OptionsPrivate.ClearPicks()
       WeakAuras.PickDisplay(pendingPickData.id, pendingPickData.tabToShow)
     end
+    OptionsPrivate.Private.Threads:Add("history_update", coroutine.create(function()
```

Comment:
this is the meat of the change, I've a suspicion that doing the work to create the archive is weirdly taxing to the game (possibly due to extreme GC churn?).